bot/tests/test_workflow.py

```diff
@@ -16,6 +16,7 @@
 from code_review_bot.tasks.default import DefaultTask
 from code_review_bot.tasks.infer import InferTask
 from code_review_bot.tasks.lint import MozLintTask
+from code_review_bot.tasks.upload import DocUploadTask
```

Comment:
Similarly here, you'll actually need to write some tests, not just import the classes